src/sqlfluff/core/config.py

```diff
@@ -1058,6 +1058,35 @@ def from_string(
             plugin_manager=plugin_manager,
         )
 
+    @classmethod
+    def from_strings(
+        cls,
+        *config_strings: str,
+        extra_config_path: Optional[str] = None,
+        ignore_local_config: bool = False,
+        overrides: Optional[Dict[str, Any]] = None,
+        plugin_manager: Optional[pluggy.PluginManager] = None,
+    ) -> "FluffConfig":
```

Comment:
I wonder if all of these could return `FluffConfig` not `"FluffConfig"`. Maybe with `from __future__ import annotations`?

Reply:
ohh good idea 👍